src/System Application/App/AppSource Gallery/src/AppSourceProductManager.Codeunit.al

```diff
@@ -15,100 +15,117 @@ using System.RestClient;
+    procedure GetCountryLetterCode(): Code[2]
     var
-        entraTenant: Codeunit "Azure AD Tenant";
+        EntraTenant: Codeunit "Azure AD Tenant";
     begin
-        exit(entraTenant.GetCountryLetterCode());
+        exit(EntraTenant.GetCountryLetterCode());
     end;
 
-    procedure AzureAdTenant_GetPreferredLanguage(): Text
+    procedure GetPreferredLanguage(): Text
     var
-        entraTenant: Codeunit "Azure AD Tenant";
+        EntraTenant: Codeunit "Azure AD Tenant";
     begin
-        exit(entraTenant.GetPreferredLanguage());
+        exit(EntraTenant.GetPreferredLanguage());
     end;
 
-    procedure AzureADTenant_GetAadTenantID(): Text
+    procedure GetAadTenantID(): Text
     var
-        entraTenant: Codeunit "Azure AD Tenant";
+        EntraTenant: Codeunit "Azure AD Tenant";
     begin
-        exit(entraTenant.GetAadTenantID());
+        exit(EntraTenant.GetAadTenantID());
     end;
 
-    procedure AzureKeyVault_GetAzureKeyVaultSecret(SecretName: Text; var Secret: SecretText);
+    procedure GetAzureKeyVaultSecret(SecretName: Text; var Secret: SecretText);
     var
         KeyVault: Codeunit "Azure Key Vault";
     begin
         KeyVault.GetAzureKeyVaultSecret(SecretName, Secret);
     end;
 
-    procedure EnvironmentInformation_GetApplicationFamily(): Text
+    procedure GetApplicationFamily(): Text
     var
         EnvironmentInformation: Codeunit "Environment Information";
     begin
         exit(EnvironmentInformation.GetApplicationFamily());
     end;
 
-    procedure EnvironmentInformation_IsSaas(): boolean
+    procedure IsSaas(): boolean
     var
         EnvironmentInformation: Codeunit "Environment Information";
     begin
         exit(EnvironmentInformation.IsSaas());
     end;
 
-    procedure Language_GetFormatRegionOrDefault(FormatRegion: Text[80]): Text
+    procedure GetFormatRegionOrDefault(FormatRegion: Text[80]): Text
     var
         Language: Codeunit Language;
     begin
         exit(Language.GetFormatRegionOrDefault(FormatRegion));
     end;
 
-    procedure RestClient_GetAsJSon(var RestClient: Codeunit "Rest Client"; RequestUri: Text): JsonToken
+    procedure GetAsJSon(var RestClient: Codeunit "Rest Client"; RequestUri: Text): JsonToken
     begin
         exit(RestClient.GetAsJSon(RequestUri));
     end;
 
-    procedure UserSettings_GetUserSettings(UserSecurityId: Guid; var TempUserSettingsRecord: record "User Settings" temporary)
+    procedure GetUserSettings(UserSecurityId: Guid; var TempUserSettingsRecord: record "User Settings" temporary)
     var
         UserSettings: Codeunit "User Settings";
     begin
         UserSettings.GetUserSettings(Database.UserSecurityID(), TempUserSettingsRecord);
     end;
 
     #endregion
-    procedure SetDependencies(SpecificDependencies: Interface "IAppSource Product Manager Dependencies")
+    procedure SetDependencies(SpecificDependencies: Interface "AppSource Product Manager Dependencies")
```

Comment:
this still seems like something you only use for testing... and its a bad practice to have these backdoors for testing only.